### ISSUE TITLE
IDE crashes when the app designer is in use

### ISSUE THREAD

**Issue by Nelrik** [01org/appframework#920]
The intel XDK IDE crashes when i change to designer tab, after i change, appears a message that says "For security reasons, framing is not allowed" in continue i click "Ok" but the IDE turn into a web browser and i can't with my job.

This is the message...
![image](https://cloud.githubusercontent.com/assets/19697672/16495823/a201abda-3eb4-11e6-9109-2c0679f4959d.png)

And this is the next step after the message
![image](https://cloud.githubusercontent.com/assets/19697672/16495867/d75f357c-3eb4-11e6-9a3a-c70881b0df7c.png)

**Comment by mesysadm** [comment]
I am also facing the same problem..

**Comment by tanganellibrasil** [comment]
I am also facing the same problem.

**Comment by Nelrik** [comment]
Some news?, someone has found the the reason of the error?

**Comment by ideaopen** [comment]
the same problem for me

**Comment by rogerioxavier** [comment]
The same. Someone has found this solution?

**Comment by Nelrik** [comment]
yes, I have noticed the same reason of the crashes, i still working without designer tab and all works fine, i think is caused by wrong configuration on contextual tab in the bottom of designer tab.

previously this tab shows the docs of a control now shows the GitHub page....

Here's a screenshot
![image](https://cloud.githubusercontent.com/assets/19697672/16770421/83f65994-4812-11e6-9b1a-a3f1f0b594c0.png)

**Comment by thiagomota456** [comment]
I am also facing the same problem.
someone found the solution?

**Comment by Nelrik** [comment]
We all are at the same situation
